src/redux/sagas/saga.js

```diff
@@ -51,7 +51,7 @@ function* fetchImages(ID) {
 }
 
 function* addImagesAsync(action) {
-    const img = yield fetchImages(action.ID);
+    const img = yield fetchImages(action.albumID);
     yield put({type: 'ADD_IMAGE_ASYNC', img});
 }
 
```

Comment:
1- Try to split your code into different files (AddAlbum, AddImage)
2- Functions like rootSaga belongs to index files

Reply:
Done.